docs/content/topics/project-developers/projects.md

```diff
@@ -9,296 +9,267 @@ aliases:
   - /topics/project-developers/projects.md
 ---
 
-TODO: update per v2
-
 # Managing Projects in Brigade
 
-In Brigade, a project is just a special Kubernetes secret. The Brigade project currently offers
-two methods to create a project: via the `brig` cli and via the [brigade-project][brigade-project-chart]
-Helm chart.  The latter is managed in the [brigadecore/charts][charts] repo
-and an in-depth overview of its configuration can be seen in the chart
-[README](https://github.com/brigadecore/charts/blob/master/chart/brigade-project/README.md).
+In Brigade, a project is a conceptual grouping of event subscriptions paired
+with logic expressing how to handle inbound events.
```

Comment:
This is a total nit and please feel free to disagree with me.

If we were to change "inbound events" to "those events," _I think_, that would more clearly connect the dots-- i.e. that the logic is for the handling of those events to which the project has subscribed.

Reply:
Clearer for sure; updated!